portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -31,20 +32,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     UserService userService = UserServiceFactory.getUserService();
     PrintWriter out = response.getWriter();
 
-    final String USERNAME_FIELD = getUsername(userService.getCurrentUser().getUserId());
-    final String COMMENT_TEXT_FIELD = request.getParameter("comment");
-    final long TIMESTAMP_FIELD = System.currentTimeMillis();
+    final String usernameField = getUsername(userService.getCurrentUser().getUserId());
+    final String commentTextField = request.getParameter(COMMENT_TEXT_KEY);
+    final long timestampField = System.currentTimeMillis();
```

Comment:
I don't think you need these anymore.  You are only using them once (below) and could just inline the calls to getUsername, getParameter, and currentTimeMillis right where they are needed.

Reply:
Inlined